### PR TITLE
Add CFT docs

### PR DESCRIPTION
**Pull request by Xenorith**
porting contents of https://www.alluxio.io/products/aws/aws-cft-tutorial/ into docs

### CODE REVIEW

File: docs/en/cloud/AWS-CFT.md [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+
+  {% endcollapsible %}
+{% endaccordion %}
+
+### Configure cluster
+
+{% accordion configureCluster %}
+  {% collapsible Configure cluster %}
+Specify the details of your Alluxio cluster in this page:
+- Stack details: Name of your cluster
```
Comment (LuQQiu) [inline docs/en/cloud/AWS-CFT.md]: Stack details is the page of holding all the CFT parameters.
The first value that need to put in is `Stack Name`: Name of your cluster.

File: docs/en/cloud/AWS-CFT.md [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+
+- Network Configuration: 
+  - Choose the [VPC and Subnet](https://docs.aws.amazon.com/vpc/latest/userguide/VPC_Subnets.html) from the existing values.
+  The default values should be available to select.
+  If not, create them following the [AWS VPC documentation](https://docs.aws.amazon.com/vpc/latest/userguide/working-with-vpcs.html).
+  These parameters isolate the cluster from other virtual private clouds, AWS regions, and availability zones.
+  - Choose at least [security group](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-network-security.html)
```
Comment (LuQQiu) [inline docs/en/cloud/AWS-CFT.md]: Choose at least `one`?

File: docs/en/cloud/AWS-CFT.md [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+  - **AlluxioProperties** can be set to provide additional Alluxio site properties.
+  Alluxio CFT only provides the necessary parameters to create an Alluxio cluster.
+  If you desire to fine-tune Alluxio behavior, specify the desired properties in the format of `KEY1=VALUE1,KEY2=VALUE2`.
+  The specified key-value pairs will be appended to the `alluxio-site.properties` file in all nodes inside the Alluxio cluster.
+
+- Advanced Alluxio Configuration: These optional parameters are only available in the advanced template.
+  - **AlluxioJournalSize** determines the EBS volume size, in GB, where Alluxio's embedded journal will write to.
+  Each Alluxio master mounts a EBS volume dedicated for logging all metadata changes to achieve fault tolerance.
+  The journal size should be proportional to the estimated magnitude of Alluxio metadata operations.
+  - **AlluxioWorkerMemPercent** is the percentage of the instance's total memory to allocate for the Alluxio worker process.
```
Comment (LuQQiu) [inline docs/en/cloud/AWS-CFT.md]: This memory is not allocated for the Alluxio worker process, but use as the Alluxio worker storage to store Alluxio cached data. 
So may be remove the process? or change it to storage.

File: docs/en/cloud/AWS-CFT.md [hunk 4]
```diff
@@ -0,0 +1,261 @@
...
+  By default this is set to 70%.
+  This space is used for caching remote data in order to drastically increase overall I/O throughput.
+  - **AlluxioWorkerSSDSize** determines the SSD volume size, in GB, to attach to each worker instance.
+  An EBS volume is attached for workers to attach remote data if the value is non-zero.
+  A SSD drive is a simple scalable alternative to instance memory to help increase the Alluxio worker's caching capacity.
+  - **AlluxioMetadataBackupDirectory** determines the path to backup Alluxio's journal contents.
+  This path is relative to Alluxio's root mount point.
+  Backup files can be use to relaunch an Alluxio cluster 
+  - **AlluxioMetadataBackupTime** is the time to schedule the daily backup operation.
+  If set to an empty string, the backup operation will be disabled.
```
Comment (LuQQiu) [inline docs/en/cloud/AWS-CFT.md]: **AlluxioMetadataBackupDirectory** decides whether backup operation is taken instead of the `AlluxioMetadataBackupTime`

**Comment by Xenorith** [comment]
i originally omitted some screenshots but we want parity with the corresponding tutorial on alluxio.io so i added back all the screenshots. @LuQQiu can you take a look again

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Pull request merged**